intranet/static/themes/afd2016/reactions.js

```diff
@@ -29,7 +29,9 @@ $(function() {
     giveReaction = function(icon) {
         var react = icon.attr("data-reaction");
         var outer = icon.parent().parent();
-        $.post(outer.attr("data-endpoint"), {"reaction": react}, function(res) {
+        $.post(outer.attr("data-endpoint"), {
+            "reaction": react
+        }, function(res) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['res' is defined but never used.](https://www.codacy.com/app/tjcsl/ion/file/2755451194/issues/source?bid=3176160&fileBranchId=3355598#l34)